Code/skyrim_ui/src/app/components/settings/settings.component.html

```diff
@@ -112,6 +112,19 @@ <h2>{{ 'COMPONENT.SETTINGS.DEBUG' | transloco }}</h2>
         >
       </div>
     </div>
+    <div class="section">
+      <h2>{{ 'COMPONENT.SETTINGS.FONT' | transloco }}</h2>
+      <div class="flex-item">
+        <label for="font-size">{{ 'COMPONENT.SETTINGS.FONT_SIZE' | transloco }}</label>
+        <span class="fontsize">
+          <button id="decrement" (click)="onFontSizeDown()"></button>
+            <input id="font-size" type="number" min=10 value={{fontSize}} max=20 step=1 [ngModel]="fontSize" 
```

Comment:
Use binding instead of interpolation

Reply:
Why an input?

Reply:
Replaced with a text element now since it's just supposed to dispaly a value. Used input before because I'm not very familiar with Angular.